### PR TITLE
Correct orthographic mistakes.

### PR DESCRIPTION
**Pull request by pkreissl**

### CODE REVIEW

File: doc/tutorials/python/01-lennard_jones/01-lennard_jones.tex [hunk 1]
```diff
@@ -129,7 +129,7 @@ \section{Introduction}
 
 \section{Background}
 Today's research on Soft Condensed Matter has brought the needs for having a 
-flexible, extensible, reliable and efficient (parallel) molecular simulation 
+flexible, extensible, reliable, and efficient (parallel) molecular simulation
```
Comment (KaiSzuttor) [inline doc/tutorials/python/01-lennard_jones/01-lennard_jones.tex]: i think theres no need for a commata here.

Comment (pkreissl) [inline doc/tutorials/python/01-lennard_jones/01-lennard_jones.tex]: This is a `serial comma', see: [https://www.oxforddictionaries.com/words/comma#commas_in_lists]

Comment (rbardak) [inline doc/tutorials/python/01-lennard_jones/01-lennard_jones.tex]: It's ambigious on whether or not to use the oxford comma for adjectives. It's not necessary in these cases, but one should advocate consistency.

**Comment by RudolfWeeber** [comment]
I think, this can be merged

**Pull request merged**